test/LibYear.Tests/Output/Json/JsonOutputTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using LibYear.Core;
+using LibYear.Core.Tests;
+using LibYear.Output.Json;
+using NSubstitute;
+using Spectre.Console;
+using Spectre.Console.Testing;
+using Xunit;
+
+namespace LibYear.Tests.Output.Json;
+
+public class JsonOutputTests
+{
+	[Fact]
+	public void NoResultsProducesNoOutput()
+	{
+		//arrange
+		var console = Substitute.For<IAnsiConsole>();
+		// Act
```

Comment:
Sorry for the formatting comments, I should really figure out how to get a linter/formatter automated. Can we keep all the arrange/act/asserts lowercase, and have a blank line before this one?